metaflow/plugins/kubernetes/kubernetes_job.py

```diff
@@ -53,12 +56,432 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def _make_domain_name(
+    jobset_name, main_job_name, main_job_index, main_pod_index, namespace
+):
+    return "%s-%s-%s-%s.%s.%s.svc.cluster.local" % (
+        jobset_name,
+        main_job_name,
+        main_job_index,
+        main_pod_index,
+        jobset_name,
+        namespace,
+    )
+
+
+class KubernetesJobSet(object):
```

Comment:
can you port this to `kubernetes_jobset.py` so that it is easy to maintain going forward?